### PR TITLE
adding some underlined text and eventually pissing Coopydood and Gentoo users off

### PR DESCRIPTION
**Pull request by eversiege**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,10 +63,12 @@ Optional dependencies can enhance your experience, such as using your VM in a GU
 
 #### Oh, and you NEED Linux.
 
-Shocker; KVM is a module built into the *Linux kernel*. Not **Windows Subsystem for Linux** or some UNIX-like terminal. You need a full install of at least base Linux **on your host**. Don't try any of that VM inception shenanigans.
+Shocker; KVM is a module built into the *Linux kernel*, <ins>not **Windows Subsystem for Linux** or some UNIX-like terminal</ins>. You need a <ins>full install</ins> of at least base Linux **on your host**. Don't try any of that VM inception shenanigans.
 
 The easiest way to do this is by grabbing some mainstream Linux distro, like **Ubuntu, Linux Mint, Manjaro, endeavourOS** - among many, many others. You can pick any one you like. If it's Linux - you can use this project. *I use Arch BTW.*
 
+Please submit an Issue
```
Comment (Coopydood) [inline README.md:70]: Rogue line (copy and paste mistake?)

File: README.md [hunk 2]
```diff
@@ -144,13 +146,13 @@ Alongside the tutorial and explanation documents, this project includes troubles
 
 You can search for your problem and find solutions in the [**Issues and Troubleshooting**](https://github.com/Coopydood/ultimate-macOS-KVM/wiki/Issues-and-Troubleshooting) section of the project wiki.
 
-If an unexpected problem with the ULTMOS project itself arises, it may be a bug, and can be reported [here](https://github.com/Coopydood/ultimate-macOS-KVM/issues/new).
+If an unexpected problem with the ULTMOS project itself arises, it may be a bug, and should be reported [here](https://github.com/Coopydood/ultimate-macOS-KVM/issues/new).
```
Comment (Coopydood) [inline README.md:149]: Unnecessary word change

File: README.md [hunk 3]
```diff
@@ -144,13 +146,13 @@ Alongside the tutorial and explanation documents, this project includes troubles
...
 
-As you use it to generate your personal files, having to re-clone the entire repo yourself while preserving your files would be a right pain in the backside. Therefore, there's an automated updater script built right-in that you can use to safely update in-place to newer versions of this project, without affecting any of your personal config files, virtual hard drives, or anything else not part of the project files. And, if an update dramatically changes directory structures from your current version, the updater automatically disables its in-place update mechanism to prevent data loss.
+As you use it to generate your personal files, having to re-clone the entire repo yourself while preserving your files would be a right pain in the backside. Therefore, there's an automated updater script built right in that you can use to safely update in-place to newer versions of this project, without affecting any of your personal config files, virtual hard drives, or anything else not part of the project files. And, if an update dramatically changes directory structures from your current version, the updater automatically disables its in-place update mechanism to prevent data loss.
```
Comment (Coopydood) [inline README.md:155]: Good hyphenation

File: README.md [hunk 4]
```diff
@@ -166,11 +168,11 @@ This is my way of giving back to the QEMU, KVM, and VFIO community. Please don't
 
 - I'm not responsible for any time you waste using this project.
 - I'm not responsible if you make an oopsie whoopsie.
-- Do NOT run anything as ``sudo``. If you have an urge to then you need to break that habit _**asap.**_ None of my scripts require superuser privileges.
+- Do NOT run anything as ``sudo``. If you have an urge to then you need to break that habit _**asap.**_ Most of my scripts do not require superuser privileges, however, the ones that do require them are mentioned.
```
Comment (Coopydood) [inline README.md:171]: Needs work; change to "the ones that do are clearly marked with a yellow ⚠️ next to the operation requiring such permissions."

File: README.md [hunk 5]
```diff
@@ -166,11 +168,11 @@ This is my way of giving back to the QEMU, KVM, and VFIO community. Please don't
...
 - Expect headaches. Some severe.
-- You need a LOT of patience. I mean it. Despite me trying to alleviate some of the hassle, you WILL run into [stupid gotchas](https://github.com/Coopydood/ultimate-macOS-KVM/wiki/Gotchas) that require trial and error out of my control.
+- You need a LOT of patience. I mean it. Despite me trying to alleviate some of the hassles, you <ins>can and will</ins> run into [stupid gotchas](https://github.com/Coopydood/ultimate-macOS-KVM/wiki/Gotchas) that require trial and error out of my control.
```
Comment (Coopydood) [inline README.md:173]: Unnecessary change

File: README.md [hunk 6]
```diff
@@ -166,11 +168,11 @@ This is my way of giving back to the QEMU, KVM, and VFIO community. Please don't
...
 - I'm by no means an expert on this stuff nor would I claim to be.
-- I've simply made *what I had to go through* **easier** for you. Hopefully.
+- I've simply made *what <ins>I</ins> had to go through **easier** for you*. Hopefully.
```
Comment (Coopydood) [inline README.md:175]: Unnecessary change

File: README.md [hunk 7]
```diff
@@ -200,4 +202,6 @@ While I am the creator of these automation/ease-of-use scripts, this project is
 
 ***
 
-<center><img src="https://github.com/Coopydood/ultimate-macOS-KVM/assets/39441479/39d78d4b-8ce8-44f4-bba7-fefdbf2f80db" width="15%"></img></center>
\ No newline at end of file
+<p align="center">
+  <img src="https://github.com/Coopydood/ultimate-macOS-KVM/assets/39441479/39d78d4b-8ce8-44f4-bba7-fefdbf2f80db" width="10%"> </img>
```
Comment (Coopydood) [inline README.md:206]: Good alignment

File: README.md [hunk 8]
```diff
@@ -14,9 +14,9 @@ Helping you build the ultimate macOS virtual machine, powered by KVM.
 
 Tired of all the restraints and poor performance of macOS VMs in hypervisors like VirtualBox or VMware? Well, the story changes when you run your virtual machines in **kernel space**. Welcome to the world of **K**ernel **V**irtual **M**achines.
 
-You might be new to QEMU/KVM, or a long-time veteran - either way this project aims to help you build a macOS virtual machine that can take full advantage of the power of KVM - but in a user-friendly and approachable way.
+You might be new to QEMU/KVM, or a long-time veteran - either way, this project aims to help you build a macOS virtual machine that can take full advantage of the power of KVM - but in a user-friendly and approachable way.
```
[the remark this answers is not in the document]
Comment (eversiege) [reply README.md]: Grammarly says otherwise

**Comment by eversiege** [comment]
Commit fixes issues; please see attached

**Review by Coopydood — COMMENTED** [review]
The "good hyphenation" comments were because I liked the changes, please reimplement them before I merge

**Comment by Coopydood** [comment]
And remove Gentoo comment

File: README.md [hunk 9]
```diff
@@ -101,8 +101,8 @@ $ git clone https://github.com/Coopydood/ultimate-macOS-KVM
 
 Alternatively, you can download the latest release here: [Download](https://github.com/Coopydood/ultimate-macOS-KVM/archive/refs/heads/main.zip)
 
-Okay, so you've cloned my repo, and cd'd into the directory. Great!
-Now, before running the script - and to avoid the subsequent and inevitable gotcha - you should enable the ``libvirtd`` daemon first if it hasn't been already. Here's an example for Arch-based systems:
+Okay, so you've cloned my repo, and `cd`'d into the directory. Great!
+Now, before running the script - and to avoid the subsequent and inevitable gotcha - you should enable the ``libvirtd`` daemon first if it hasn't been already. Here's an example for Arch-based systems (and pretty much most Linux distros):
```
[the remark this answers is not in the document]
Comment (eversiege) [reply README.md:105]: h

**Review by Coopydood — APPROVED** [review]
Approved
